ansible_base/authentication/utils/claims.py

```diff
@@ -346,52 +346,47 @@ def create_organizations_and_teams(results) -> None:
     Use the results data from 'create_claims' to make the Organization
     and Team objects necessary for the user if they do not exist.
     """
-
     # a flat list of relevant org names
-    org_list = set()
+    org_set = set()
     # a flat list of relevant org:team names
-    team_list = set()
-
+    team_org_set = set()
     # a structure for caching org+team,member info
     membership_map = {}
 
     # fill in the top level org membership data ...
     for org_name, is_member in results['claims']['organization_membership'].items():
-        if is_member:
-            org_list.add(org_name)
-            membership_map[org_name] = {'id': None, 'teams': {}}
+        if not is_member:
```

Comment:
I don't find this more readable, it's creating 4 lines from 3 lines, moreover introducing negative condition ( which is not recommended)
This would be useful if the following code will contain more lines, with more indentation

Reply:
clarified during meeting, anyway this was more comment than request - resolving